pkg/server/problem_ranges.go

```diff
@@ -289,7 +289,7 @@ const debugProblemRangesTemplate = `
         <DIV CLASS="row">
           {{- if $.LeaderNotLeaseholder}}
             {{- range $_, $r := $.LeaderNotLeaseholder}}
-              <a href="/debug/range?id={{$r}}">{{$r}}</a>
+              <a href="/debug/range?id={{$r}}">{{$r}}</a> &nbsp;
```

Comment:
There's both a breaking and a non-breaking space here (and the line is getting broken). I think what you want is to remove the whitespace-stripping `-` from the beginning of the `{{- end}}` directive (unless you intend for there to be two spaces between entries in this list).